index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h3>Hello World, we are software developers!</h3>
+
+      <div id="main">
+        <h3 id="title">Dr.Norman Borlug</h3>
+          <div id="img-div">
+            <!---img here-->
+            <img src="https://img.freepik.com/free-photo/pleased-young-female-doctor-wearing-medical-robe-stethoscope-around-neck-standing-with-closed-posture_409827-254.jpg?w=2000" alt="doctor" id="image">
+            <div id="image-caption">Caption Here</div>
```

Comment:
I suggest wrapping the elements in a section tag. This is so as to improve its semantic element and thus describes the contents in the section.